packages/backend/src/assets/ai.json

```diff
@@ -69,6 +69,23 @@
       ],
       "backend": "llama-cpp"
     },
+    {
+      "id": "chatbot-java",
+      "description": "This is a Java-based chat demo application.",
+      "name": "Java-based ChatBot",
+      "repository": "https://github.com/containers/ai-lab-recipes",
+      "ref": "main",
```

Comment:
Not sure what to put here until it's part of a release, so I used `main` for now.

Reply:
We would prefer to have a fixed ref here. You can use the full commit ID of the current HEAD for this

Reply:
Ok @feloy, done